rock.osdeps

```diff
@@ -265,7 +265,9 @@ python-pexpect:
     opensuse: python-pexpect
 
 python-msgpack:
-    debian,ubuntu: python-msgpack
+    debian,ubuntu:
+        "18.04,18.10,19.04,19.10": python-msgpack
```

Comment:
Could you please add 16.04 here as well?

Reply:
Done

Reply:
Thanks